Navbars/Side_Navbar_02/code/style.css

```diff
@@ -1,96 +1,103 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
 body {
     background-color: #dcdcdc;
-    font-family: Arial, Helvetica, sans-serif;
-    font-size: 20px;
 }
```

Comment:
Don't target the body directly anyone using this component might have other options and choice . So, fix it